mlir/test/Transforms/canonicalize-debuginfo.mlir

```diff
@@ -2,7 +2,7 @@
 
 // CHECK-LABEL: func @merge_constants
 func.func @merge_constants() -> (index, index, index, index) {
-  // CHECK-NEXT: arith.constant 42 : index loc(#[[UnknownLoc:.*]])
```

Comment:
Why is this check next broken? Can you provide the before/after output for this test?

Reply:
Check next is not broken and the output for this function is identical before and after. 

I see that the CHECK-NEXT is a meaningful assertion here so reverted back to it.